ccmlib/scylla_repository.py

```diff
@@ -155,6 +165,7 @@ def setup(version, verbose=True):
     s3_url = ''
     type_n_version = version.split(':', 1)
     scylla_product = os.environ.get('SCYLLA_PRODUCT', 'scylla')
+    scylla_arch = os.environ.get('SCYLLA_ARCH', 'x86_64')
```

Comment:
is this env variable set? Should it be passed/set to dtest docker?

Reply:
Not necessarily, currently we are targeting only x86_64, but when we would want to target arm, we should add it there.